build.gradle

```diff
@@ -56,6 +56,7 @@ compileTestJava {
 dependencies {
   compileOnly "io.vertx:vertx-core:$vertxVersion"
   compile "fr.wseduc:web-utils:$webutilsVersion"
+  compile "org.entcore:common:$entCoreVersion"
```

Comment:
Dans l'idée, les modules techniques doivent pouvoir être utilisé dans des applications qui ne sont pas l'ENT. Il ne faut pas que des modules techniques dépendent d'entcore.

Reply:
C'est noté. La dépendance à entcore a bien été supprimée.